login/token_context/token.go

```diff
@@ -0,0 +1,32 @@
+// Package token_context contains the code that extract token manager from the
+// context.
+package token_context
```

Comment:
The package is named token_context which is descriptive of what is done inside. I didn't want to confuse it with token pkg